spa/assets/worlds/post/vrml/post.wrl

```diff
@@ -3357,30 +3381,137 @@ texCoordIndex [
 }
 ]
 }
-DEF Sound01 Transform {
-translation -23.56 0 0.0001179
-rotation 0 -1 0 -1.571
+
+#audio file missing smile07
+#DEF Sound01 Transform {
+#translation -23.56 0 0.0001179
+#rotation 0 -1 0 -1.571
+#children [
+#DEF Sound01 Sound {
+#direction 0 0 1
+#intensity 5
+#location 0 0 0
+#maxBack 10
+#maxFront 10
+#minBack 1
+#minFront 1
+#priority 0
+#spatialize TRUE
+#source
+#DEF AudioClip01 AudioClip {
+#description ""
+#url "citizendir.wav"
+#pitch 1
+#loop FALSE
+#stopTime 1
+#}
+#}
+#]
+#}
+#ROUTE TouchSensor01-SENSOR.touchTime TO	AudioClip01.set_startTime
+
+#door fix smile07
+Group {
+children [
+DEF doors Transform {
+#doors will open
+translation 0 9.53674e-007 0
+children [
+DEF l TouchSensor {
+enabled IS enabled
+touchTime IS touch_changed
+}
+###insert door geometry here
+Transform {
+rotation 0 1 0 -1.57
+translation 36.154 2 4
 children [
-DEF Sound01 Sound {
-direction 0 0 1
-intensity 5
-location 0 0 0
-maxBack 10
-maxFront 10
-minBack 1
-minFront 1
-priority 0
-spatialize TRUE
-source
-DEF AudioClip01 AudioClip {
-description ""
-url "citizendir.wav"
-pitch 1
-loop FALSE
-stopTime 1
+Shape {
+appearance Appearance {
+texture ImageTexture {
+url "door.jpg"
+repeatS TRUE
+repeatT TRUE
+}
+textureTransform TextureTransform {
+translation .306 0
+scale 6 2
+}
+}
+geometry IndexedFaceSet {
+coord DEF k Coordinate {
+point [
+-.269 -.3 -4, -2.1 -.3 -4, -2.1 2.99 -4, -.269 2.99 -4, 1.69 2.99 -4,
+1.69 -.3 -4, -.269 -.3 -4, -.269 2.99 -4,
+]
 }
+coordIndex [
+0 1 2 3 -1, 4 5 6 7 -1,
+]
+colorPerVertex FALSE
+texCoord TextureCoordinate {
+point [
+.526 0, .368 0, .369 .529, .527 .529, .685 .529, .684 0,
+]
 }
+ccw FALSE
+creaseAngle 3.14
+texCoordIndex [
+0 1 2 3 -1, 4 5 0 3 -1,
 ]
 }
+}
+DEF Clock TimeSensor {
+startTime -1
+cycleInterval .5
+}
+DEF n CoordinateInterpolator {
+key [
+0 .5 1
+]
+keyValue [
+-.269 -.3 -4 -2.1 -.3 -4 -2.1 2.99 -4 -.269 2.99 -4 1.69 2.99 -4 1.69 -.3 -4 -.269
+-.3 -4 -.269 2.99 -4 -.269 -.3 -4.07 -2.1 -.3 -4.07 -2.1 2.99 -4.07 -.269 2.99
+-4.07 1.69 2.99 -4.07 1.69 -.3 -4.07 -.269 -.3 -4.07 -.269 2.99 -4.07 -1.78 -.3
+-4.07 -3.2 -.3 -4.07 -3.2 2.99 -4.07 -1.78 2.99 -4.07 2.62 2.99 -4.07 2.62 -.3
+-4.07 1.19 -.3 -4.07 1.19 2.99 -4.07
+]
+}
+DEF Beam Script{
+eventIn SFBool Timer
+url "javascript:
+function Timer(value){
+if (!value){
+window.open('/#/place/enter','_SELF');

```

Comment:
@smile0711 this will open the plaza in a new window. Ideally, we should keep the user in the same window. I suggest replacing this with `window.location.href`

Reply:
The "_SELF" parameter keeps the user in the current window, but will change it to window.location.assign() if we would prefer and see if it still works.